setup.cfg

```diff
@@ -8,6 +8,8 @@ packages=find:
 install_requires =
     pytest >= 6.2.2
     pytest-cov >= 2.11.1
+    redis >= 3.5.3
+    Flask >= 1.1.2
```

Comment:
I know that @rileyalankirk changed the project from a `setup.py` to a `setup.cfg` in https://github.com/cs334s21/capstone2021/pull/8.

What is the relationship between `install_requires` in the `.cfg` file and the `requirements.txt` file?  We have the list of libraries duplicated, which is dangerous when we inevitably forget to add a new library to both.

As I understand it, when you pip install our packages (`c21server`, etc.) it will automatically install the dependencies from `install_requires`.  If this is true, then we can probably eliminate the `requirements.txt` - thus removing the redundant list.

Can someone please test this?   Assuming this works as I described, we can remove the `requirements.txt` in a separate PR.  This will be cleaner than adding more to this one.

So, can we get rid of `requirements.txt`

Reply:
> I know that @rileyalankirk changed the project from a `setup.py` to a `setup.cfg` in #8.
> 
> What is the relationship between `install_requires` in the `.cfg` file and the `requirements.txt` file? We have the list of libraries duplicated, which is dangerous when we inevitably forget to add a new library to both.
> 
> As I understand it, when you pip install our packages (`c21server`, etc.) it will automatically install the dependencies from `install_requires`. If this is true, then we can probably eliminate the `requirements.txt` - thus removing the redundant list.
> 
> Can someone please test this? Assuming this works as I described, we can remove the `requirements.txt` in a separate PR. This will be cleaner than adding more to this one.
> 
> So, can we get rid of `requirements.txt`

I have tested this; installing our packages will first install the dependencies from `install_requires`. We do not need `requirements.txt` and thus we can remove it from #13. Similar to requirements.txt, we would need to reinstall within our virtual environment if we add new dependencies after our initial installation.